### PR TITLE
attempt 1

### PR DESCRIPTION
**Pull request by Lamle104**

### CODE REVIEW

**Comment by Lamle104** [comment]
Do i need to install PIL?

**Comment by halfaro1** [comment]
@Lamle104 you need to install Pillow as it's discussed in the Chapter -- did you do that?

File: resizeAndAddLogo.py [hunk 1]
```diff
@@ -8,7 +9,28 @@
 logoWidth, logoHeight = logoIm.size
 
 #TODO: Loop over all files in the working directory
+for filename in os.listdir('.'):
```
Comment (halfaro1) [inline resizeAndAddLogo.py]: @Lamle104 this currently looks in the current working directory, but your image files are in the `originals` directory

**Comment by halfaro1** [comment]
@Lamle104 
Thanks for continuing to work on this project! Unfortunately, I have to grade this project as-is. I am closing out this pull request and entering your grade on Blackboard. Please see your grade there.

**Pull request closed without merging**